rio/src/RIO/List/Partial.hs

```diff
@@ -17,15 +17,8 @@ module RIO.List.Partial
   , Data.List.maximumBy
   , Data.List.minimumBy
 
-  -- * Building lists
-
-  -- ** Scans
-  , Data.List.scanl1
```

Comment:
For backwards compat, these should remain here, but a comment explaining that they should be removed in the future would be a good thing.